index.html

```diff
@@ -11,11 +11,98 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
+
 <body>
+    <main>
+    <!-- top - header section / navbar + logo + unordered list of links that do not open-->        
+<header class="header">
+    <div class="logo">
+        <img class="header-logo" src="img/karma-logo.svg" alt="karma logo"> 
+    </div>
+    <nav class="nav">
+        <div class="links">
+        <ul>
+            <li>
+                <a class="nav-bold" href="">Meet Karma</a>
+            </li>
+            <li>
+                <a class="nav-link" href="">How It Works</a>
+            </li>
+            <li>
+                <a class="nav-link" href="">Store</a>
+            </li>
+            <li>
+                <a class="nav-link" href="">Blog</a>
+            </li>
+            <li>
+                <a class="nav-link" href="">Help</a>
+            </li>
+            <li>
+                <a class="nav-link" href="">Login</a>
+            </li>
+        </ul>             
+        </div>       
+    </nav>
+</header>
+<!-- Section 1 - intro section + button -->
+
+<section class="hero-header">
+    <div class="hero-desc">
+            <h1>Introducing Karma</h1>
+            <h2>Bring WiFi with you, everywhere you go.</h2>
+        <button class="hero-btn">Learn More</button>    
+    </div>    
+</section>
 
-    <!-- Add your HTML markup here -->
+<!-- information section + 3 favicon icons-->
+<section class="section-grid"
+    <div class="grid">
+        <div class="desc">
+            <h1>Everyone needs a little Karma.</h1>          
+        </div>        
+        <div class="icon">
+            <img class="info" src="img/icon-devices.svg" alt="devices icon"></div>
+        <div class="desc">
+            <h3>Internet for all devices</h3>  
+        </div>
+        <div class="icon">
+            <img class="info" src="img/icon-coffee.svg" alt="coffee icon">
+        </div>
+        <div class="desc">
+            <h3>Boost your productivity</h3>
+        </div>
+        <div class="icon">
+            <img class="info" src="img/icon-refill.svg" alt="pay as you go refill icon">
+        </div>
+        <div class="desc">
+            <h3>Pay as You Go</h3>
+        </div>
+</section>
+</div>
+<!-- a grey line above the footer -->
+<hr>
+<!-- footer section content + social media logos -->
+<footer>
+    <div class="social-media">
```

Comment:
Good stucture here - but do you need this extra div? You could move the `social-media` class to the footer, and get rid of the div